src/components/map/MapService.js

```diff
@@ -562,6 +570,17 @@ goog.require('ga_urlutils_service');
                     '/home.html'
               };
 
+              // Luftfahrthindernis
+              if (response.data['ch.bazl.luftfahrthindernis']) {
+                response.data['ch.bazl.luftfahrthindernis'].config3d =
+                    'ch.bazl.luftfahrthindernis.3d';
+                response.data['ch.bazl.luftfahrthindernis.3d'] = {
+                  type: 'kml',
+                  url: 'https://dav0.bgdi.admin.ch/bazl_web/bern/' +
+                      'Active_Obstacles_18.04.2016.bern.kml'
+                };
+              }
+
```

Comment:
Why specific date?

In any case, we have to verify with data owner, before we can use their kml directly. There was a time when this kml was not really correct. But I think it's a very good selling point. Just keep it in mind.